### ISSUE TITLE
Error Connecting: The specified Power BI workspace is not found

### ISSUE THREAD

**Issue by miguel-s-nascimento** [DaxStudio/DaxStudio#1290]
When trying to connect to a workspace of type "word1 - word2 word3" it gets an error saying it is not found. 

I have tried to parse the whitespaces as %20 and the hifen as %2D and combinations with these 2 and still couldn't connect.

![image](https://github.com/user-attachments/assets/98560164-6aef-4129-8556-a51c485d6ffb)

**Comment by dgosbell** [comment]
Did you copy the connection details from the license page in your workspace?
![image](https://github.com/user-attachments/assets/e51f0489-a791-4ce7-a368-7391ad4b528a)
I created a workspace with a similar name and it connected without error
![image](https://github.com/user-attachments/assets/22e86c32-822c-40db-afea-f6dc3dd4a72e)

The error above is also coming from the Power BI service so it could also be a permissions error. Can you access this workspace from the web portal?

**Comment by dgosbell** [comment]
No, whitespace and hyphens in workspaces names are not an issue I use them all the time.

**Comment by Hanko-Swart** [comment]
We are actually experiencing the same since this morning
